components/CreateWorkout/CreateWorkout.jsx

```diff
@@ -129,7 +144,40 @@ export default ({ navigation }) => {
               };
             }),
           };
-
+          let muscleGroups = itemState[0].muscleGroups; 
```

Comment:
Do we need two different objects for redux and database? I think they can we the same right?